src/pytest_cov/engine.py

```diff
@@ -70,6 +70,10 @@ def __init__(self, cov_source, cov_report, cov_config, cov_append, cov_branch, c
         self.topdir = os.getcwd()
         self.is_collocated = None
 
+        # If unset (the default), indicate fallback behavior for other files like pyproject.toml.
+        # See https://github.com/nedbat/coveragepy/blob/6.2/coverage/control.py#L144-L146
+        self.config_file_choice = self.cov_config or True
```

Comment:
Also I'd simply rename this to config_file. Maybe it's just me but a dropdrown widget pops up in my head when I see "choice" - too much Django I guess xD